internal/nginx/manager.go

```diff
@@ -268,7 +268,7 @@ func (lm *LocalManager) Start(done chan error) {
 }
 
 // Reload reloads NGINX.
-func (lm *LocalManager) Reload() error {
+func (lm *LocalManager) Reload(isEndPointUpdate bool) error {
```

Comment:
Could you use one away to name this parameter across all methods for consistency? Better to use plural as this is how the object is named in Kubernetes. Also, in the rest of the code, endpoint related ids don't use mixed caps (no `endPoint`).  `isEndpointsUpdate` would be a good name